### PR TITLE
Add settings components into layout

### LINKED ISSUE #1570: Move existing settings components into settings view

### LINKED ISSUE #1576: Settings menu transition

### PR DESCRIPTION
**Pull request by gabaldon**
This pr add the settings' components into the settings layout. It also creates a transition for the settings view.
close #1570 
close #1576

### CODE REVIEW

**Comment by aesedepece** [comment]
When this is ready for review, please feel free to assign it to me!

File: src/App.vue [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <template>
   <div id="app">
-    <router-view />
+    <transition :name="transitionName">
```
Comment (aesedepece) [inline src/App.vue:3]: Please change the transition name to something else than `transitionName` :stuck_out_tongue_winking_eye:

Comment (aesedepece) [reply src/App.vue:3]: Ah ok never mind, now I understand that this is catching the transition name from a variable named liked that hehehe, sorry!

**Comment by aesedepece** [comment]
My personal wishlist regarding this:
- Rename any mention of "currency" as "unit", e.g. "default unit"
- Rename "wit" → "Wit" in list of units
- Center settings menu so it's shown centered in wider window sizes
- Create a larger active area that expands at least 10px around the settings button in the unlock screen, so it is easier to click it. A positive padding, plus a negative margin, do normally the trick.

**Review by aesedepece — APPROVED** [review]
Fantastic!

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "sheikah-witnet-wallet",
   "productName": "Sheikah Witnet wallet",
-  "version": "1.6.0",
+  "version": "1.5.0",
```
Comment (tmpolaczyk) [inline package.json:4]:
```suggestion
  "version": "1.6.0",
```

**Review by aesedepece — APPROVED** [review]

**Review by aesedepece — APPROVED** [review]

**Review by aesedepece — APPROVED** [review]

**Review by aesedepece — APPROVED** [review]

**Review by aesedepece — APPROVED** [review]

**Pull request merged**